Makefile

```diff
@@ -53,6 +53,7 @@ style: reports pipenv
 
 	-$(PIPENV) run mypy . --check-untyped-defs | tee -a reports/mypy.log || echo "mypy failed" >> reports/mypy_errors.log
```

Comment:
`|| echo "mypy failed" >> reports/mypy_errors.log` is obsolete then

Reply:
Thanks, makes sense!